WPMedia/fusion-news-theme-blocks
# THEMES-759: Identity Blocks | Log in form

## Description

This PR adds in the Log In form Identity block. As more components have been added to the Identity block, I've brought the naming and organization inline with our other blocks.

## Jira Ticket

- [THEMES-759](https://arcpublishing.atlassian.net/browse/THEMES-759)

## Acceptance Criteria

The Identity Log In - Arc Block is using the v2 architecture.

## Test Steps

1. Checkout this branch `git checkout THEMES-759`
2. Run fusion repo with linked blocks `npx fusion start -f -l @wpmedia/identity-block`
3. Create a test page using the Single Column layout.
4. Place the "Identity Log In - Arc Block" on the page.
5. The designs of the log in form should match the Figma docs (https://www.figma.com/file/vWVGgZfZVBdSX7ACo3kmjr/Enhanced-Styling-Design-System?type=design&node-id=12874-178005&mode=design&t=JptOC1IZ0G9wCmAD-4).
    * Note that items below the login button in the design doc are part of blocks yet to be converted to v2.
    * Once the account management block is converted, testing the full functionality of the sign up and log in process will be possible.

## Author Checklist

        please explain why (so that we can fix it whenever it gets refactored).

## Reviewer Checklist

_The reviewer of the PR should copy-paste this template into the review comments on review._



[THEMES-759]: https://arcpublishing.atlassian.net/browse/THEMES-759?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
